accord-core/src/test/java/accord/burn/TopologyUpdates.java

```diff
@@ -185,46 +186,47 @@ private static Collection<Node.Id> allNodesFor(Txn txn, Topology... topologies)
-            getUninterruptibly(node.commandStores().forEach(commandStore -> InMemoryCommandStore.inMemory(commandStore).forCommittedInEpoch(ranges, srcEpoch, commandConsumer)));
+            start = node.commandStores().forEach(commandStore -> InMemoryCommandStore.inMemory(commandStore).forCommittedInEpoch(ranges, srcEpoch, commandConsumer));
         else
-            getUninterruptibly(node.commandStores().forEach(commandStore -> InMemoryCommandStore.inMemory(commandStore).forEpochCommands(ranges, srcEpoch, commandConsumer)));
+            start = node.commandStores().forEach(commandStore -> InMemoryCommandStore.inMemory(commandStore).forEpochCommands(ranges, srcEpoch, commandConsumer));
 
-        return syncMessages.entrySet().stream().map(e -> {
+        return start.map(ignore -> syncMessages.entrySet().stream().map(e -> {
             CommandSync sync = new CommandSync(e.getKey(), e.getValue(), srcEpoch, trgEpoch);
             return MessageTask.of(node, recipients.apply(sync), sync.toString(), sync::process);
-        });
+        }));
     }
 
     private static final boolean PREACCEPTED = false;
     private static final boolean COMMITTED_ONLY = true;
 
-    /**
-     * Syncs all replicated commands. Overkill, but useful for confirming issues in optimizedSync
-     */
-    private static Stream<MessageTask> thoroughSync(Node node, long syncEpoch) throws ExecutionException
-    {
-        Topology syncTopology = node.configService().getTopologyForEpoch(syncEpoch);
-        Topology localTopology = syncTopology.forNode(node.id());
-        Function<CommandSync, Collection<Node.Id>> allNodes = cmd -> node.topology().withUnsyncedEpochs(cmd.route, syncEpoch + 1).nodes();
-
-        Ranges ranges = localTopology.ranges();
-        Stream<MessageTask> messageStream = Stream.empty();
-        for (long epoch=1; epoch<=syncEpoch; epoch++)
-        {
-            messageStream = Stream.concat(messageStream, syncEpochCommands(node, epoch, ranges, allNodes, syncEpoch, COMMITTED_ONLY));
-        }
-        return messageStream;
-    }
+//    /**
+//     * Syncs all replicated commands. Overkill, but useful for confirming issues in optimizedSync
+//     */
+//    private static Stream<MessageTask> thoroughSync(Node node, long syncEpoch) throws ExecutionException
+//    {
+//        Topology syncTopology = node.configService().getTopologyForEpoch(syncEpoch);
+//        Topology localTopology = syncTopology.forNode(node.id());
+//        Function<CommandSync, Collection<Node.Id>> allNodes = cmd -> node.topology().withUnsyncedEpochs(cmd.route, syncEpoch + 1).nodes();
+//
+//        Ranges ranges = localTopology.ranges();
+//        Stream<MessageTask> messageStream = Stream.empty();
+//        for (long epoch=1; epoch<=syncEpoch; epoch++)
+//        {
+//            messageStream = Stream.concat(messageStream, syncEpochCommands(node, epoch, ranges, allNodes, syncEpoch, COMMITTED_ONLY));
+//        }
+//        return messageStream;
+//    }
```

Comment:
why is this commented out?

Reply:
left a comment about this, I did this and left GH comment; this is dead code and the related logic had a dead lock issue with this [patch](https://issues.apache.org/jira/browse/CASSANDRA-18203).  I am 100% fine making this async, but left it there to have that convo.

This is also from https://issues.apache.org/jira/browse/CASSANDRA-18203 and not this ticket, so you wouldn't see that comment here.

Reply:
Got it, yeah I didn't see the comment. I'll be a little pedantic and say that although the code is unreachable, it does provide value to the project as a tool when debugging some burn test failures, and is therefore not "dead". However, it also isn't run and is therefore quickly stops working if not being used diagnosing failures. Having said that, I'm not sure if the value it provides is still more than the cost to maintain it.

Reply:
in this context the maintain was super simple, so cost is low.  Now does this function "work" is unknown...  I don't mind keeping it now that it compiles